client/components/vertical-nav/item/style.scss

```diff
@@ -1,3 +1,6 @@
+@import 'assets/stylesheets/shared/_colors';
```

Comment:
After some discussion we decided to take approach of adding explicit imports. It looks like we can use codemod provided by Yury to make it work.